src/styles/sass/blocks/card.scss

```
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  padding: 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  font-size: $base-font-size;
+  font-weight: 400;
+  border-radius: 5px;
+
+  @include smooth-animation;
+
+  &__img-container {
+    margin-top: 16px;
+    padding: 0 3px;
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__section {
+    margin-top: 16px;
+    display: flex;
+  }
+
+  &__section--description {
+    margin-top: 0;
+    flex-direction: column;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
```

```suggestion
    margin-bottom: 4px;
```
but it's better to use only top or only bottom margins